inc/parserutils.php

```diff
@@ -148,7 +148,7 @@ function p_cached_output($file, $format='xhtml', $id='') {
     } else {
         $parsed = p_render($format, p_cached_instructions($file,false,$id), $info);
 
-        if ($info['cache'] && $cache->storeCache($parsed)) {              // storeCache() attempts to save cachefile
+        if (isset($info['cache']) && $info['cache'] && $cache->storeCache($parsed)) {              // storeCache() attempts to save cachefile
```

Comment:
`!empty($info['cache']` would make more sense and would also keep the maximum line length in check